spec.yaml

```diff
@@ -1395,22 +1395,39 @@ paths:
         Candles are aggregated across all markets for the base currencies, which necessitates converting to a common
         quote currency. Nomics converts all markets into USD in order to aggregated candles.
 
-        Candles have the following history based on size:
-
-        * `1d`: From inception updated every 15 minutes
-        * `1h`: Rolling 30 days updated every 5 minutes
+        Historical candles are available for both intervals from
+        inception.  However, hourly aggregated candles have a 7 day
+        limit on the length of the date range you can request if you
+        request a date range whose start timestamp is older than 30
+        days.
```

Comment:
Having this information in a paragraph will make it easy to miss. Could we do this instead?

```markdown
Candles have the following time range limits based on interval and time:

* `1d`: Available from inception with no range limit
* `7d`: Available from inception. Time range limits:
  * Past 30 days: accessible in a single request
  * Older than 30 days: 7 days at a time
```

Reply:
Done! Thanks for the suggestion.